pretyping/evarconv.ml

```diff
@@ -1231,7 +1241,8 @@ let apply_on_subterm env evd fixedref f test c t =
         try test env !evdref k c t
```

Comment:
By the way, @mattam82 is is normal that the `evd` value returned by `test` above is quietly dropped?

Reply:
No, I actually just fixed that yesterday in the unifall-apply branch, Maxime will soon push a PR fixing this.